pkg/collectors/builder.go

```diff
@@ -212,6 +216,54 @@ func (b *Builder) buildGroupCollector() *collector.Collector {
 	return collector.NewCollector(store)
 }
 
+func (b *Builder) buildInfrastructureCollector() *collector.Collector {
+	filteredMetricFamilies := metric.FilterMetricFamilies(b.whiteBlackList, infrastructureMetricFamilies)
```

Comment:
nit: we should change the name `whiteBlackList`

Reply:
This change was made already in upstream kube-state-metrics, where this comes from, and should be in the 2.0 release.